test/expected.txt

```diff
@@ -135,6 +137,9 @@
 || 
 || 5 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  sort]
+|| Address book has been sorted!
+|| ===================================================
```

Comment:
Good work updating the test, but looking at this it's hard to tell if `sort` is really working. It would be better if we start off with a list that's not already sorted.